plugins/chat/assets/javascripts/discourse/services/chat-drafts-manager.js

```diff
@@ -0,0 +1,25 @@
+import Service from "@ember/service";
+import ChatMessage from "discourse/plugins/chat/discourse/models/chat-message";
+export default class ChatDraftsManager extends Service {
+  drafts = {};
```

Comment:
I wonder if we split this into `threadDrafts` and `channelDrafts`? Or are we just thinking we will only have one draft per channel, including threads?

Reply:
Then we could have more specific APIs like `addChannelDraft(channelId, message)` etc.